x-pack/plugins/apm/server/lib/fleet/register_fleet_policy_callbacks.ts

```diff
@@ -91,16 +92,23 @@ function registerAgentConfigExternalCallback({
       ruleDataClient,
     });
     const agentConfigurations = await listConfigurations({ setup });
-    return getPackagePolicyWithAgentConfigurations(
-      packagePolicy,
-      agentConfigurations
-    );
+    const artifacts = await listArtifacts({ fleetPluginStart });
+    return {
+      ...getPackagePolicyWithAgentConfigurations(
+        packagePolicy,
+        agentConfigurations
+      ),
+      ...getPackagePolicyWithSourceMap({
+        packagePolicy,
+        artifacts,
+      }),
```

Comment:
I think the results of the second function would omit updates from the first function and overwrite it with old values. Probably best to compose these functions:
```
getPackagePolicyWithAgentConfigurations(getPackagePolicyWithSourceMap(packagePolicy, artifacts), agentConfigurations)
```